inlong-manager/manager-service/src/main/java/org/apache/inlong/manager/service/core/impl/ConsumptionServiceImpl.java

```diff
@@ -418,6 +402,12 @@ private void fullConsumptionInfo(ConsumptionInfo info) {
                 streamTopics.forEach(stream -> topicSet.remove(stream.getMqResource()));
                 Preconditions.checkEmpty(topicSet, "topic [" + topicSet + "] not belong to inlong group " + groupId);
             }
+            InlongGroupEntity inlongGroupEntity = groupMapper.selectByGroupId(groupId);
+            if (null != inlongGroupEntity) {
+                info.setTopic(String.format(InlongGroupSettings.PULSAR_TOPIC_FORMAT, "public",
```

Comment:
The default tenant is `public`, but before the default value, we should query from the cluster first.

Please refer to the `InlongClusterService#getOne` to get more information.